### PR TITLE
Backend dropdown fix

### PR DESCRIPTION
**Pull request by coolcat-creations**
The dropdowns are cut off at the bottom of page when appearing on the end of the site. 
Example: mod_smart_search

bildschirmfoto 2015-06-16 um 11 57 05

To test please apply this patch an test if it works in different browsers and the other dropdowns work as expected.

Thanks!

### CODE REVIEW

**Comment by brianteeman** [comment]
From my test on this your solution worked for me in Safari and chrome on OSX

As you have edited the template.css then you will need to make the change in the less file <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="http://issues.joomla.org/tracker/joomla-cms/7182">issues.joomla.org/joomla-cms/7182</a>.</sub>

File: administrator/templates/isis/less/template.less [hunk 1]
```diff
@@ -584,6 +584,11 @@ html[dir=rtl] .quick-icons .nav-list [class^="icon-"], html[dir=rtl] .quick-icon
 .chzn-drop {
 	max-width: 100% !important;
 }
+/*Fix for last dropdown */
+.control-group:last-of-type .chzn-container-active .chzn-drop  {
+  position:static!important
```
Comment (Fedik) [inline administrator/templates/isis/less/template.less]: some people will hate you :wink:
please try avoid use of `!important`

**Comment by infograf768** [comment]
Any reason for not adding the classes to media/jui/bootstrap-extended.less and then run generatecss.php ?

**Comment by designbengel** [comment]
To be honest just my lack of knowledge how things work together in the structure. So should i remove it from isis and put it into bootstrap-extended.less, yes?

**Comment by mbabker** [comment]
> Any reason for not adding the classes to media/jui/bootstrap-extended.less and then run generatecss.php ?

Is the change something that needs to be made to all templates or is this a template specific fix?  If the former, you have a case for using the extended LESS file.  If it's specific to the template, this is fine.

**Comment by astridx** [comment]
It works for me with Firefox (38.0.5), IE (11.0.20) and Chrome (43.0.2357.125) on Windows 7.
But it worked only in the Module Manager. 

I try to open a drop down in Article Manager | Edit Article |Options and here the bottom was still cut off.
<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="http://issues.joomla.org/tracker/joomla-cms/7182">issues.joomla.org/joomla-cms/7182</a>.</sub>

**Comment by Worti2** [comment]
It works fine on Firefox 38.0.5 and Windows 7 and IE 11.0.9600.17843 on Windows 7. It works in the Article Manager, too.

**Comment by smz** [comment]
Tested this, and it works (*and it is a welcome fix!*), but, sorry, I'm not going to give this my official positive test until those `!important` declarations are there. After all, forgive me the the pun, they don't seems to be important... :smile: 

Also, in `/media/com_finder/css/finder.css ` a last blank line seems to be missing (*Isn't this amongst our CS requirements? Am I wrong? Is it for PHP only? Travis not catching this?*)

**Comment by Niranjan-J007** [comment]
Tested on Opera 30.0.1835.125 / Win 10 Insider Preview Build 10074 #jquery15 <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="http://issues.joomla.org/tracker/joomla-cms/7182">issues.joomla.org/joomla-cms/7182</a>.</sub>

**Comment by gunjanpatel** [comment]
I have applied patch and tested successfully in #jquery15 hackathon. We got it fixed. But there are other pages which is having issue. For Example: Frontend Editing Page. Not sure that should be fixed in the same PR.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="http://issues.joomla.org/tracker/joomla-cms/7182">issues.joomla.org/joomla-cms/7182</a>.</sub>

**Comment by gunjanpatel** [comment]
Ideally, we should move it RTC but as this issue is not fixed in other pages then not sure should I move it to RTC or wait for the fixes or should not be counted as a successful test (Test is successful as described)<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="http://issues.joomla.org/tracker/joomla-cms/7182">issues.joomla.org/joomla-cms/7182</a>.</sub>

**Comment by ramya185** [comment]
Tested on chrome 43.0.2357.134 #jqconf15

Steps followed: 
- Go to module manager
- Click on new
- Select Smart search
- The Set Item drop down still remains in the cut state.screen shot 2015-07-25 at 02 53 46<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="http://issues.joomla.org/tracker/joomla-cms/7182">issues.joomla.org/joomla-cms/7182</a>.</sub>

**Comment by infograf768** [comment]
Works fine here on Macintosh, all browsers.

**Comment by mflr26** [comment]
My test on Chrome (W8) was successfully.
First i checked the code fragment before applying it, then after applying it.
After recognizing the change of code, i tested the view at the backend.
First the dropdown has been cut off, afterwards it was correctly displayed behind the bar.
<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="http://issues.joomla.org/tracker/joomla-cms/7182">issues.joomla.org/joomla-cms/7182</a>.</sub>

**Comment by infograf768** [comment]
Is not that solved already by https://github.com/joomla/joomla-cms/pull/7609 ?

**Comment by Teck95** [comment]
Tested with smart search and worked (Windows 8 Waterfox).
Small issue is:
If the dropdown-menu is open, it's not possible to close it again
by clicking on the button opening it. You have to click on the site outside
the dropdown menu.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="http://issues.joomla.org/tracker/joomla-cms/7182">issues.joomla.org/joomla-cms/7182</a>.</sub>

**Comment by brianteeman** [comment]
As stated above this appears to have been fixed elsewhere. Can you please confirm if it has indeed been fixed (looks like it to me) and then we can close this.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/7182">issues.joomla.org/joomla-cms/7182</a>.</sub>

**Comment by roland-d** [comment]
Hello @designbengel

Thank you for your contribution.

The last comment here was on November 18th. So the question is, Is this issue/pull request still valid? 
If no reply is received within 4 weeks we will close this issue.

Thanks for understanding!<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/7182">issues.joomla.org/joomla-cms/7182</a>.</sub>

**Comment by designbengel** [comment]
Hello @roland-d 
The Issue still exists in 3.4.6 but i´ll take a look at 3.5 next days and let you know :) thanks!<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/7182">issues.joomla.org/joomla-cms/7182</a>.</sub>

**Comment by designbengel** [comment]
Issue seems fixed in 3.5 :)<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/7182">issues.joomla.org/joomla-cms/7182</a>.</sub>

**Comment by roland-d** [comment]
@designbengel So we can close this. I will close this PR, if it needs further attention it can always be opened later on. Thank you all for your contributions.

**Pull request closed without merging**